### ISSUE TITLE
The loss may be negative and not converge

### ISSUE THREAD

**Issue by eedavidwu** [lehaifeng/DASNet#8]
It seems, the loss cannot converge after lots of epoches...(So the dataset and the net may have sth wrong). 
Any suggestions about the data?...
Also, the loss has some negative value (which is a bit strange).

**Comment by lehaifeng** [comment]
Do you use DContrastive loss for training?There may be problems with the advanced version of pytorch.Please check your version

**Comment by TingquanGao** [comment]
> It seems, the loss cannot converge after lots of epoches...(So the dataset and the net may have sth wrong).
> Any suggestions about the data?...
> Also, the loss has some negative value (which is a bit strange).

Hi, did you solve your problem? I encountered the same problem: the loss value may be negative and not converge.
